docs/activiti-alfresco-content.service.md

```diff
@@ -0,0 +1,126 @@
+# Activiti Alfresco Content Service
+
+Get Alfresco Repository folder content based on a Repository account configured in Alfresco Process Services (APS). 
+It is possible to configure multiple Alfresco Repository accounts in APS (i.e. multiple Alfresco Servers).
+This service can also be used to link Alfresco content as related content in APS. 
+Content such as documents and other files can be attached to Process Instances 
+and Task Instances as related content.
+
+<aside class="warning">
+At the moment you must provide the `ActivitiAlfrescoContentService` class from your `NgModule` for it to work:
+```ts
+@NgModule({
+...
+  providers: [ActivitiAlfrescoContentService] 
+})
+```
+And also import it in the way shown below.
+</aside>
+
+## Importing
+
+```ts
+import { ActivitiAlfrescoContentService } from 'ng2-activiti-form/src/services/activiti-alfresco.service';
```

Comment:
+import { ActivitiAlfrescoContentService } from 'ng2-activiti-form' should be enough

Reply:
I filed a Jira about it: https://issues.alfresco.com/jira/browse/ADF-1891, a providers declaration is also necessary to get it to work.